lab_1_classify_by_unigrams/main.py

```diff
@@ -106,6 +155,27 @@ def detect_language(
     In case of corrupt input arguments, None is returned
     """
 
+    if (not isinstance(unknown_profile, dict) and not isinstance(profile_1, dict)
+            and not isinstance(profile_2, dict)):
+        return None
+    if (unknown_profile is None or profile_1 is None
+            or profile_2 is None):
+        return None
+    if (isinstance(unknown_profile, dict) and isinstance(profile_1, dict)
+            and isinstance(profile_2, dict)):
+        mse_prof_1 = compare_profiles(unknown_profile, profile_1)
+        mse_prof_2 = compare_profiles(unknown_profile, profile_2)
+        if mse_prof_1 is not None and mse_prof_2 is not None:
+            prof_1_name = profile_1.get('name')
+            prof_2_name = profile_2.get('name')
+            if isinstance(prof_1_name, str) and isinstance(prof_2_name, str):
+                if mse_prof_1 < mse_prof_2:
+                    return prof_1_name
+                if mse_prof_1 > mse_prof_2:
+                    return prof_2_name
+                return min(prof_1_name, prof_2_name)
```

Comment:
can you explain why are you using `min()` here?

Reply:
If mse_prof_1 = mse_prof_2 then it'll return the profile which name will be the first in the alphabet order